chart/values.yaml

```diff
@@ -22,6 +22,17 @@ distributor:
   config:
     queueGroup:
       enabled: true                          # Enable connection via Nats queue group to support exactly-once message processing
+  securityContext:                           # Default security context of the distributor; DO NOT CHANGE!
+    privileged: false
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 1234
+    runAsGroup: 2345
```

Comment:
Can you explain those numbers for the security context of distributor?

For reference, this is what Keptn core is using: https://github.com/keptn/keptn/blob/979e81daa1803f2b21069ba12274fe24275968ad/installer/manifests/keptn/charts/control-plane/templates/_helpers.tpl#L248-L264

Reply:
The distributor uses now the same `securityContext` as Keptn core, it can also be overwritten in a similar way